controllers/v1/learningResources.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * name : learningResources.js
+ * author : Rakesh Kumar
+ * created-date : 24-Jun-2020
+ * Description : Related to learning resources
+ */
+
+const learningResourceshelper = require(MODULES_BASE_PATH + "/learningResources/helper.js");
+
+/**
+   * LearningResources
+   * @class
+*/
+module.exports = class LearningResources {
+
+  static get name() {
+    return "learningResources";
+  }
+
+  /**
+  * @api {get} /sunbird/api/v1/learningResources/list?limit=10&page=1 
+  * To get list of learning resources
+  * @apiVersion 1.0.0
+  * @apiGroup Learning Resources
+  * @apiHeader {String} X-authenticated-user-token Authenticity token
```

Comment:
@rakeshSgr - Let this be available only if the internal access token is provided.